spec/opportunities/org_admin_creates_opportunity_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'rails_helper'
+
+feature "user creates an opportunity for an organization" do
+
+  let!(:user) { FactoryGirl.create(:user, volunteer: false) }
+  let!(:organization) { FactoryGirl.create(:organization) }
+  let!(:org_admin) { FactoryGirl.create(
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.